storagenode/storagenodedb/orders.go

```diff
@@ -16,38 +16,46 @@ import (
 	"storj.io/storj/storagenode/orders"
 )
 
-type ordersdb struct{ *InfoDB }
+// ErrOrders represents errors from the ordersdb database.
+var ErrOrders = errs.Class("ordersdb error")
 
-// Orders returns database for storing orders
-func (db *DB) Orders() orders.DB { return db.info.Orders() }
+type ordersdb struct {
```

Comment:
nit - maybe this should be camelcase since `pieceExpirationDB` uses camelcase and uppercases DB? 🤷🏻‍♀️

Reply:
Done!